python/pyspark/pandas/frame.py

```diff
@@ -1847,14 +1665,14 @@ def corrwith(
         ----------
         other : DataFrame, Series
             Object with which to compute correlations.
-
+        axis : int, default 0 or 'index'
+            Can only be set to 0 at the moment.
         drop : bool, default False
             Drop missing indices from result.
-
-        method : str, default 'pearson'
-            Method of correlation, one of:
-
+        method : {'pearson', 'spearman', 'kendall'}
```

Comment:
qq: do we also need to implement `callable` as pandas does ?
<img width="569" alt="Screen Shot 2022-09-21 at 9 06 01 AM" src="https://user-images.githubusercontent.com/44108233/191385853-2462b16c-84c4-4c37-8f71-9e3deb0bb4d6.png">

Reply:
good question, I think it's a bit hard to support this `callable`:
it takes two arrays, so should collect all values in the columns, but it's not scalable then

maybe, we can support another `callable`: `Callable[[Column, Column], float]`, which is an aggregation function, this may make sense. I think we need more discussion/thoughs on it.